Tribler/Core/APIImplementation/LaunchManyCore.py

```diff
@@ -680,29 +680,35 @@ def early_shutdown(self):
         self.shutdownstarttime = timemod.time()
         if self.boosting_manager:
             yield self.boosting_manager.shutdown()
-            self.boosting_manager = None
+        self.boosting_manager = None
```

Comment:
I don't think this is necessary, if `self.boosting_manager` fails it is already None. Same for the other cases.